### PR TITLE
Integrating casl in meshery UI

### PR DESCRIPTION
**Pull request by smartdev09**
**Notes for Reviewers**

This PR fixes #

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: ui/pages/_app.js [hunk 1]
```diff
@@ -175,6 +180,27 @@ class MesheryApp extends App {
     }
     const disposeK8sContextSubscription = k8sContextSubscription();
     this.setState({ disposeK8sContextSubscription })
+
```
[the remark this answers is not in the document]
Comment (UsmanDev09) [reply ui/pages/_app.js:183]: To persist it upon reload

File: ui/components/Can.js
Comment (theBeginner86) [inline ui/components/Can.js:1]: Should we put this under https://github.com/meshery/meshery/tree/master/ui/utils? // @aabidsofi19

File: ui/utils/permissions_keys.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+export const keys = {
+  VIEW_APPLICATIONS : {
```
Comment (theBeginner86) [inline ui/utils/permissions_keys.js:2]: Will you be adding more changes to ensure we enforce these keys in the UI?

**Review by theBeginner86 — COMMENTED** [review]
I wonder how this would work for local provider 🤔
As we are not generating keys for local provider, at least till now.

File: ui/components/Can.js
Comment (theBeginner86) [reply ui/components/Can.js:1]: also please consider using small case for naming files.

Comment (aabidsofi19) [reply ui/components/Can.js:1]: I think this should go in contexts/can.js .

**Comment by leecalcote** [comment]
@UsmanDev09 has stepped away and is no longer responding here. 

@aabidsofi19 will you take this initiative forward by either completing the work in this branch or by closing this PR and starting afresh?

**Comment by aabidsofi19** [comment]
@leecalcote I will look into this

**Pull request closed without merging**